.github/workflows/release-pipeline.yml

```
@@ -130,15 +130,15 @@ jobs:
 
       - name: Bump version and push tag
         id: tag_version
-        uses: mathieudutour/github-tag-action@v6.2
+        uses: mathieudutour/github-tag-action@a22cf08638b34d5badda920f9daf6e72c477b07b # v6.2
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           custom_tag: ${{ github.event.inputs.tag_version }}
```

actionlint (1.7.12)

[error] property "tag_version" is not defined in object type {default_bump: string; latest: string; operator_version: string; replace_operator_version: string; supported_openshift_range: string}

(expression)